client/assets/images/intent-screen/build.svg

```diff
@@ -0,0 +1,12 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think we don't need `build.svg` and `write.svg` because I put it into `icons.tsx`